src/armTemplates/resources/appServicePlan.ts

```diff
@@ -71,10 +77,15 @@ export class AppServicePlanResource implements ArmResourceTemplateGenerator {
       ...config.provider.storageAccount,
     };
 
+    const { os } = config.provider;
+
     const params: AppServicePlanParams = {
       appServicePlanName: {
         value: AppServicePlanResource.getResourceName(config),
       },
+      kind: {
+        value: (os === FunctionAppOS.LINUX) ? "Linux" : undefined,
```

Comment:
interesting - so undefined implies Windows?

Reply:
The armService removes parameters with undefined parameters on deployment. So by keeping it undefined, it just won't be added